pkg/sdn/plugin/node.go

```diff
@@ -124,7 +133,24 @@ func (node *OsdnNode) Start() error {
 }
 
 func (node *OsdnNode) GetLocalPods(namespace string) ([]kapi.Pod, error) {
-	return node.registry.GetRunningPods(node.hostName, namespace)
+	fieldSelector := fields.Set{"spec.host": node.hostName}.AsSelector()
```

Comment:
that's the same field (deprecated version and current version), though the assumption that it is a hostname has always been tenuous.